wercker.yml

```diff
@@ -5,11 +5,14 @@ services:
 build:
     steps:
         - virtualenv:
-            install_wheel: false
+            install_wheel: true
```

Comment:
Wheels are just an "evolved" pip. They are indeed faster, but many packages currently don't have wheels (see http://pythonwheels.com/). That's why I left it on false.